### PR TITLE
Dockerfile: Update ubuntu base

### PR DESCRIPTION
**Pull request by mrueg**
Updated ubuntu base image used to build from 16.04 to 22.04 as 16.04 is out of (unpaid) support

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:xenial
+FROM ubuntu:20.04
```
Comment (AkihiroSuda) [inline Dockerfile:1]: Now there is 22.04

Comment (onedr0p) [inline Dockerfile:1]:
```suggestion
FROM ubuntu:22.04
```

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Comment by mrueg** [comment]
Repository owner doesn't seem to be interested in this change.

**Comment by onedr0p** [comment]
I've switched to https://github.com/openSUSE/catatonit 
Pretty much the same thing but looks more active.

**Pull request closed without merging**